src/sokoban.cpp

```diff
@@ -21,6 +24,26 @@ bool Sokoban::solved() {
     return true;
 }
 
-std::vector<std::string> Sokoban::board() {
+std::vector<std::string> Sokoban::board() 
+{
     return _board;
+}
+
+
+void Sokobann::locate_player()
```

Comment:
This is a pretty good method name, but `set_player_location` might be a bit more precise, emphasizing that we're modifying state.

We also have a typo, `Sokobann`, suggesting that the code wasn't executed prior to the PR. This won't compile.